### DISCUSSION TITLE
AIA features and ACPI supported on RISCV Virt

### DISCUSSION THREAD

**Discussion by pandianAKmcw** [tianocore/edk2#4667] [Q&A]
Hi, I am working on booting EDK-II on RISCV architecture. And I have compiled RISCV Virt binary under the OVMF directory. This binary is used with QEMU to boot to UEFI shell and then kernel.

Could you please mention what are the AIA(Advanced Interrupt Architecture) features and ACPI(Advanced Configuration and Power Interface) tables supported from the UEFI side.. It would be great if you could provide the list (or) commandline to check. ACPI is switched off for the virt board. This is the command used:

qemu-system-riscv64 -nographic  -machine acpi=off\
-drive file=<path>/RISCV_VIRT.fd,if=pflash,format=raw,unit=1 \
-machine virt -m 2G \
-drive file=<root_fs_path>,format=raw,id=hd0 -device virtio-blk-device,drive=hd0 \
-drive file=<built_kernel>,format=raw,id=hd1 -device virtio-blk-device,drive=hd1

**Comment by vlsunil** [comment]
Like I mentioned over email, AIA is not supported yet. The patches are under review and WIP.
